conference/react-js/src/scss/_common.scss

```diff
@@ -199,10 +199,13 @@ html {
 
     &-user {
       word-wrap: break-word;
-      font-family: "Oswald Bold";
+      font-family: "Oswald Bold", sans-serif;
       display: inline-block;
       max-width: 100%;
       }
+
+    &_small {}
```

Comment:
What do this selector?

Reply:
add margin to confirmation message.
moved margin from row 208 to this class)